### PR TITLE
user is able to add questions to database

### LINKED ISSUE #147: Backend for add questions
Backend require for the posting questions to the client-side, @purnima143

### PR DESCRIPTION
**Pull request by himanshujaidka**
closes #147 

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by himanshujaidka** [comment]
@maw1a @codewithvk mentors, pls review

File: server/src/models/questions.models.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        required: true,
+        trim: true
+    },
+    tags: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    createdBy: {
+        type: mongoose.Schema.Types.ObjectId, ref: 'User', required: true
```
Comment (codewithvk) [inline server/src/models/questions.models.js:16]: Please write the object's element in a separate line or you can formate code for more readability.

**Review by codewithvk — CHANGES_REQUESTED** [review]
Greate job @himanshujaidka ,
Please Make your code in a format so it increases readability :)

File: server/src/models/questions.models.js (hunk 1 above)
Comment (himanshujaidka) [reply server/src/models/questions.models.js:16]: okay

**Pull request merged**